pdk/1.1/modules/ROOT/pages/policies-pdk-upgrade-pdk.adoc

```diff
@@ -0,0 +1,103 @@
+= Upgrading PDK
+ifndef::env-site,env-github[]
```

Comment:
Do we need these attribute bits? I don't see them in most docs...

Reply:
All pages in Flex have them. So, I'm not 100% if we need them, but it does seem to be at least what we use.